sema/commons/aggregator/aggregator.py

```
@@ -3,38 +3,67 @@
+
+            for p in getMatchingGlobPaths(
+                self._input_path, g, makeRelative=False
+            ):
                 if p.is_file():
-                    if p.resolve() == self.output_path.resolve():
+                    if p.resolve() == self._output_path.resolve():
                         continue
                     try:
-                        self.graph.parse(p, format=fmt)
+                        self._graph.parse(p, format=fmt)
                     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)